caluma/caluma_form/structure.py

```diff
@@ -38,6 +40,10 @@ def root(self):
         return self
 
     def get(self, name, default=None):
+        for original, aliases in self.aliases.items():
```

Comment:
I think if you flip the data structure here, you might be able to get rid of the loop and improve lookup speeds. If it's of the structure `name` -> `actual_name`, you could reduce the slow lookup loop into this:

```python
name = self.aliases.get(name, name)
```